README.rst

```diff
@@ -1001,6 +1121,59 @@ validating custom parameters of a ``Check`` object.
 This setting allows you to choose whether `config_applied <#configuration-applied>`_ checks should be
 created automatically for newly registered devices. It's enabled by default.
 
+``OPENWISP_MONITORING_AUTO_IPERF``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``False``   |
++--------------+-------------+
+
+Whether `Iperf <#iperf-1>`_ checks are created automatically for devices. The devices must have ``SSH`` `access credential <https://github.com/openwisp/openwisp-controller#save-ssh-private-key-in-openwisp-access-credentials>`_  & 
```

Comment:
I don't think we need to point out here that the device needs a working SSH connection here. 

Something like this would work fine 

```
This setting allows you to choose whether `iperf <#iperf-1>`_ checks should be
created automatically for newly registered devices. It's disabled by default.
```